### ISSUE TITLE
Errors using Kopia UI with Google Drive rclone repository

### ISSUE THREAD

**Issue by MartinHammarstedt** [kopia/kopia#1698]
I upgraded to Kopia UI v0.10.3 yesterday, and have so far not been able to successfully create a snapshot on my Google Drive rclone repository. All I get is a lot of error messages like this:

```
got error Write s/fc5/d3234587f80a998c45ce4671f3ff6-s5faa1d85462feec510c.f-3481185880096948071: 400 when PutBlob(sfc5d3234587f80a998c45ce4671f3ff6-s5faa1d85462feec510c) (#0), sleeping for 100ms before retrying
```

I tried downgrading to v0.10.1 and things started working as they should again. Tried once more to upgrade, but ran into the same errors.

The task ended with the following message:

```
Error: error flushing writer: error flushing manifests: unable to write content: unable to create pending pack: unable to get session ID: unable to write session marker: unable to write session marker: sfc5d3234587f80a998c45ce4671f3ff6-s5faa1d85462feec510c: error writing blob using WebDAV: unable to complete PutBlob(sfc5d3234587f80a998c45ce4671f3ff6-s5faa1d85462feec510c) despite 10 retries, last error: Write s/fc5/d3234587f80a998c45ce4671f3ff6-s5faa1d85462feec510c.f-7690199500257153002: 400.
```

**Comment by EsEnZeT** [comment]
I confirm I have exact same issue, the last working release for me is v0.10.1

**Comment by jkowalski** [comment]
I tried this with latest code and CLI and can't repro:

```
$ kopia --config-file test.config repo create rclone --remote-path gdrive:kopia44 --password=foo
Initializing repository with:
  block hash:          BLAKE2B-256-128
  encryption:          AES256-GCM-HMAC-SHA256
  splitter:            DYNAMIC-4M-BUZHASH
Connected to repository.

NOTICE: Kopia will check for updates on GitHub every 7 days, starting 24 hours after first use.
To disable this behavior, set environment variable KOPIA_CHECK_FOR_UPDATES=false
Alternatively you can remove the file "/Users/jarek/Library/Application Support/kopia/test.config.update-info.json".

Retention:
  Annual snapshots:    3           (defined for this target)
  Monthly snapshots:  24           (defined for this target)
  Weekly snapshots:    4           (defined for this target)
  Daily snapshots:     7           (defined for this target)
  Hourly snapshots:   48           (defined for this target)
  Latest snapshots:   10           (defined for this target)
Compression disabled.

To find more information about default policy run 'kopia policy get'.
To change the policy use 'kopia policy set' command.

NOTE: Kopia will perform quick maintenance of the repository automatically every 1h0m0s
and full maintenance every 24h0m0s when running as jarek@jareks-mbp-2.

See https://kopia.io/docs/advanced/maintenance/ for more information.

$ kopia --config-file test.config snap create ./fs
Snapshotting jarek@jareks-mbp-2:/Users/jarek/Projects/Kopia/fs ...
 * 0 hashing, 21 hashed (80 KB), 0 cached (0 B), uploaded 201 B, estimated 80 KB (100.0%) 0s left 
Created snapshot with root k336a35233b3c75eb7150ea3d7f50b9e6 and ID a55423a1e74a267b9a4ed5b16f037e52 in 2s
Running full maintenance...
Looking for active contents...
  Processed 27 contents, discovered 27...
Looking for unreferenced contents...
Rewriting contents from short packs...
Not enough time has passed since previous successful Snapshot GC. Will try again next time.
Skipping blob deletion because not enough time has passed yet (59m58s left).
Cleaned up 0 logs.
Cleaning up old index blobs which have already been compacted...
Finished full maintenance.

$ kopia --config-file test.config repo disconnect
deleted repository password for /Users/jarek/Library/Application Support/kopia/test.config.

$ kopia --config-file test.config repo connect rclone --remote-path gdrive:kopia44 --password=foo
Connected to repository.

NOTICE: Kopia will check for updates on GitHub every 7 days, starting 24 hours after first use.
To disable this behavior, set environment variable KOPIA_CHECK_FOR_UPDATES=false
Alternatively you can remove the file "/Users/jarek/Library/Application Support/kopia/test.config.update-info.json".

$ kopia --config-file test.config snap ls
jarek@jareks-mbp-2:/Users/jarek/Projects/Kopia/fs
  2022-02-01 19:49:05 PST k336a35233b3c75eb7150ea3d7f50b9e6 80 KB drwxr-xr-x files:21 dirs:6 (latest-1,hourly-1,daily-1,weekly-1,monthly-1,annual-1)
```

Let's see if there's anything special about KopiaUI here.

**Comment by jkowalski** [comment]
This does appear to be UI specific. Strange.

**Comment by jkowalski** [comment]
Ok, I understand the problem - rclone is getting killed soon after opening the repository due to context cancelation, root cause is #1691 which introduced asynchronous repo initialization in KopiaUI.

Fix incoming.

**Comment by jkowalski** [comment]
This was fixed in #1713 will be in the next release.
